server/src/main/java/org/opensearch/search/profile/Timer.java

```diff
@@ -92,6 +96,14 @@ public final long getCount() {
         return count;
     }
 
+    /** Return the timer start time in nanoseconds.*/
+    public final long getTimerStartTime() {
```

Comment:
I think this property should be renamed (since timer could be used many times): 

```suggestion
    public final long getEarliestStartTime() {
```